lib/rules/web/admin_console/4.5/common_ui_elements.xyaml

```diff
@@ -368,6 +368,7 @@ click_kebab_of_one_resource:
   - selector:
       xpath: //*[text()='<resource_name>']//ancestor::tr//button[@data-test-id='kebab-button']
     op: click
+    timeout: 20
```

Comment:
@yanpzhan  Why timeout is added?

Reply:
Sometimes met failure that the kebab was no clicked open, not always.

Reply:
ok